README.md

```diff
@@ -54,6 +54,17 @@ Please, set valid commit message, to increment the right version
 ### To increase version (X.y.z) version
 `````perf: your commit message...`````
 
+# Logos
+
+- all logos should be in SVG so mobile apps wouldn't need to support 5 images of each easset that will grealy increase the app size and complexity on frontend.
+- all logos should be a fixed sized, 36x36 according figma. While it's easy to downscale an image in web using css transfrom or viewport, it would not work on mobile. Thus all assets should be already withing correct width and height. 
+
+### Libs that did NOT succeed with current assets svg
```

Comment:
unnecessary information for public repo

Reply:
It doesn't matter, it saves you time the next time we have a pain in the ass with assets, since I spent a lot of time and we'll have this conversation again when we are going to add new asssets

Reply:
I will move it to downscaler.js if it hurts your eyes in readme